File: Praktikum 4. Loop/Soal 2/PRAK402-2310817310001-AvantioFierzaPatria.py
batas_maksimal = int(input("Masukkan batas maksimal bilangan: "))

print("Urutan bilangan ganjil:", end=" ")
for i in range(1, batas_maksimal + 1, 2):
    print(i, end=" ")

print("\nUrutan bilangan genap:", end=" ")
i = batas_maksimal
while i >= 2:
    if i % 2 == 1:
        i -= 1
    print(i, end=" ")
    i -= 2

print()

File: Praktikum 4. Loop/Soal 2/PRAK402-2310817310001-AvantioFierzaPatria.c
#include <stdio.h>

int main()
{
    int batas_maksimal, i;

    printf("Masukkan batas maksimal bilangan: ");
    scanf("%d", &batas_maksimal);

    printf("Urutan bilangan ganjil: ");
    for (i = 1; i <= batas_maksimal; i += 2)
    {
        printf("%d ", i);
    }

      printf("\nUrutan bilangan genap: ");
    for (i = batas_maksimal; i >= 2; i -= 2)
    {
        if (i % 2 == 1)
        {
            i = i - 1;
        }
        printf("%d ", i);
    }

    printf("\n");
    return 0;
}
